components/FeatureHighlight/BUILD

```diff
@@ -70,6 +70,18 @@ mdc_objc_library(
     visibility = ["//visibility:public"],
 )
 
+mdc_objc_library(
+    name = "FeatureHighlightAccessibilityMutator",
+    srcs = native.glob(["src/FeatureHighlightAccessibilityMutator/*.m"]),
+    hdrs = native.glob(["src/FeatureHighlightAccessibilityMutator/*.h"]),
+    includes = ["src/FeatureHighlightAccessibilityMutator"],
```

Comment:
Please don't export this to the include path.

Reply:
Done.

Reply:
Looks like the `includes` property is still here.

Reply:
Spoke offline: This is a limitation of Bazel builds until we get full rewrite rules for ObjC files.